docker-compose.yml

```diff
@@ -165,7 +165,7 @@ services:
       - DOCKER_VOLUMES=/cache
 
   screenshotter:
-    image: lucify/screenshotter
+    image: 366857391347.dkr.ecr.eu-west-1.amazonaws.com/screenshotter:63e5b0
```

Comment:
The plain docker-compose should use public images, and the other ones should be in docker-compose-ecr.yml. 

I changed this.